src/config.cc

```diff
@@ -539,6 +538,21 @@ Status Config::finish() {
   if ((cluster_enabled) && !tokens.empty()) {
     return Status(Status::NotOK, "enabled cluster mode wasn't allowed while the namespace exists");
   }
+  if (unixsocket.empty()) {
+    if (binds.size() == 0) {
+      binds.emplace_back(kDefaultBindAddress);
+    }
+    if (port == 0) {
+      port = kDefaultPort;
+    }
```

Comment:
I think if in current config, `port == 0` and `unixsocket` is empty, kvrocks would be better to report an error and then terminate, rather than assigning a default port.

Reply:
Yes, that was my first intention. But currently, if the config is empty (I mean absolutely empty), kvrocks will start with defaults showing just the warning. If I report an error if port is 0 and unixsocket is empty, the behavior will be changed.

Reply:
Let me see..

How about
```
{"port", true, new IntField(&port, 0, 0, 65535)},
```
rather than
```
{"port", true, new IntField(&port, 6666, 0, 65535)},
```
?

Reply:
Yes, it may work. I'll check and add changes if everything is OK.

Reply:
Done